plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -564,6 +564,11 @@ func GetResourceRequest(pod *v1.Pod) *schedulercache.Resource {
 func podName(pod *v1.Pod) string {
 	return pod.Namespace + "/" + pod.Name
 }
+// As of now lists, need to convert them to sets.
```

Comment:
Can we have a single dictionary for all types of predicates instead of multiple dictionaries for different predicate types?